src/commonMain/kotlin/org/kosat/Incremental.kt

```diff
@@ -0,0 +1,5 @@
+package org.kosat
+
+interface Incremental {
```

Comment:
It's supposed to be an interface for our heuristics only, because some of them on the stage of creation doesn't work properly without maintaining incremental option. Now only one class really use it, should it be removed?